### PR TITLE
W-11517065 update archive site link

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/index.adoc
Comment (cheungaryk) [inline modules/ROOT/pages/index.adoc]: it should be https

**Review by cheungaryk — CHANGES_REQUESTED** [review]

**Review by IsaacEldridge — APPROVED** [review]

**Comment by cheungaryk** [comment]
The change looks good to me so I agree with Isaac's approval. However, I am modifying the nav grouping in the new archive site and the new archive site is not ready to be public yet (for instance, you can see 2 home icons in https://archive.docs.mulesoft.com/). This work is expected to be done early next week - until then, please do not merge.

**Review by cheungaryk — APPROVED** [review]
Please do not merge it yet, as explained in my previous comment. I can merge once the archive site is ready 😄

**Pull request merged**
